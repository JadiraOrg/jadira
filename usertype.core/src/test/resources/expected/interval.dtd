<!ELEMENT dataset (
    INTERVAL*)>

<!ELEMENT INTERVAL EMPTY>
<!ATTLIST INTERVAL
    ID CDATA #REQUIRED
    MY_START CDATA #IMPLIED
    MY_END CDATA #IMPLIED
    NAME CDATA #IMPLIED
>

