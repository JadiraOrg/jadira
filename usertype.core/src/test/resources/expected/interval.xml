<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE dataset SYSTEM "interval.dtd">
<!-- Yes we were on Summertime all year in 1970 -->
<dataset>
  <INTERVAL ID="0" MY_START="1970-01-01 01:00:00.0" MY_END="1970-01-12 18:46:40.0" NAME="test_0"/>
  <INTERVAL ID="1" MY_START="1970-01-12 14:46:40.0" MY_END="1970-01-24 04:33:20.0" NAME="test_1"/>
  <INTERVAL ID="2" NAME="test_2"/>
</dataset>